src/components/App/App.scss

```diff
@@ -0,0 +1,27 @@
+@use 'variables'
```

Comment:
I haven't used this syntax before, would love to talk about it if it has worked for you in the past.  In the branch I pulled down I changed this line to ```@import '../../variables.scss'``` (it auto corrected from _variables not sure why) and used the variables you created and it's showing the correct color on the web page.

Reply:
I added the _ before the variables file name trying to troubleshoot with the Sass docs.  Maybe we don't need it?  Thank you for looking at this and figuring it out!  I'm excited to compare notes and figure out what the issue was on my end.